src/assertions/SocialTradingAssertions.ts

```diff
@@ -157,4 +157,15 @@ export class SocialTradingAssertions {
       throw new Error(socialTradingErrors.INSUFFICIENT_TIME_PASSED(validUpdateFormattedDate));
     }
   }
+
+  public feeDoesNotExceedMaximumOnCalculator(
```

Comment:
Dont think this is necessary, you can use `assert.common.isGreaterOrEqualThan` with a custom string